code/style.css

```diff
@@ -1,7 +1,275 @@
+html {
+  font-size: 14px;
+}
+
+h1 {
+  font-size: 48px;
+}
+
+header p {
+  font-size: 28px;
+  font-weight: bold;
+}
+
+body {
+  margin: 0;
+  font-family: "Comic Sans MS", cursive, sans-serif;
+  color: #056676;
+  background-color: #a3d2ca;
+}
```

Comment:
Try to keep the html and body styling at the top of the CSS file, since they wrap all content in HTML.